#: __init__.py:96
msgid "NextGIS geoinformation system"
msgstr "Геоинформационна система NextGIS"

#: __init__.py:218
msgid "Linux kernel"
msgstr "Ядро на Linux"

#: __init__.py:221
msgid "OS distribution"
msgstr "Дистибутив на ОС"

#: __init__.py:233
msgid "CPU"
msgstr "CPU"

#: __init__.py:237
msgid "RAM"
msgstr "RAM"

#: exception.py:90
msgid "Validation error"
msgstr "Грешка при валидация"

#: exception.py:95
msgid "Forbidden"
msgstr ""

#: exception.py:100
msgid "Insufficient permissions"
msgstr "Недостатъчни права"

#: exception.py:105
msgid "Operational error"
msgstr "Операционна грешка"

#: storage.py:222
msgid "Estimation is already running"
msgstr "Оценяването е в процес"

#: test/test_exception.py:75
#, python-format
msgid "The answer is %d"
msgstr "Отговорът е %d"

